File: app/src/main/java/com/datarockets/mnchkn/activities/dashboard/DashboardPresenter.java
package com.datarockets.mnchkn.activities.dashboard;

import com.datarockets.mnchkn.models.Player;

public interface DashboardPresenter {
    void updatePlayerListItem(Player player, int position);
    void onCreate();
    void onResume();
    void onDestroy();
    void setGameFinished();
    void insertStep(Player player);
}

File: app/src/main/kotlin/com/datarockets/mnchkn/activities/dashboard/DashboardPresenter.kt
package com.datarockets.mnchkn.activities.dashboard

import com.datarockets.mnchkn.models.Player

interface DashboardPresenter {
    fun updatePlayerListItem(player: Player, position: Int)
    fun onCreate()
    fun onResume()
    fun onDestroy()
    fun setGameFinished()
    fun insertStep(player: Player)
}
